dashboard/app.py

```diff
@@ -49,6 +49,7 @@
 try:
     if os.environ["ENABLE_DASHBOARD_MONITORING"]:
         metrics = PrometheusMetrics(app)
+        metrics.start_http_server(9000)
```

Comment:
I suggest you also make this configurable so that it's not a code change each time.